airbyte-cdk/python/unit_tests/sources/streams/http/test_http.py

```diff
@@ -406,14 +398,14 @@ class CacheHttpStreamWithSlices(CacheHttpStream):
     paths = ["", "search"]
 
     def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
-        return f'{stream_slice.get("path")}'
+        return f'{stream_slice["path"]}' if stream_slice else ""
 
     def stream_slices(self, **kwargs) -> Iterable[Optional[Mapping[str, Any]]]:
         for path in self.paths:
             yield {"path": path}
 
     def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
-        yield response
+        yield {"value": len(response.text)}
```

Comment:
Just out of curiosity, why this radical change?

Reply:
it needs to be a mapping according to signature